config.json

```diff
@@ -1151,7 +1163,6 @@
     }
   ],
   "foregone": [
-    "space-age",
```

Comment:
Why was it foregone ?

Reply:
Unknown, looks like it was a while ago. @soniakeys do you remember why it was ignored in 92cfaeb99ae95e12ab0c1fbc6f673321aad4b1a0?